app/views/instrument_price_policies/_adjustment_row.html.haml

```diff
@@ -0,0 +1,21 @@
+%td
+  = hidden_field_tag "price_policy_#{price_group.id}[usage_rate]",
+    display_usage_rate(price_group, price_policy),
+    class: "usage_cost"
+  = t("price_policies.adjustment")
+  %br
+  = text_field_tag "price_policy_#{price_group.id}[usage_subsidy]",
+    display_usage_subsidy(price_group, price_policy), size: 8, class: "usage_adjustment"
+
+%td
+  = t("price_policies.amount")
+  %br
+  %span.minimum_cost{ data: { usage_subsidy: price_policy.usage_subsidy } }
+  = hidden_price_policy_tag(price_group.id, price_policy.minimum_cost, :minimum_cost, params["price_policy_#{price_group.id}"])
+
+- if local_assigns[:cancellation] != false
```

Comment:
It might be better for secure room price policies to be a completely different set of partials, but I went with this for now, since the only current difference between instrument and secure room policies is secure rooms don't have a cancelation fee. This was much quicker for now. We can revisit later if necessary.